livekit-plugins/livekit-plugins-google/livekit/plugins/google/llm.py

```diff
@@ -73,6 +75,7 @@ def __init__(
         frequency_penalty: NotGivenOr[float] = NOT_GIVEN,
         tool_choice: NotGivenOr[ToolChoice] = NOT_GIVEN,
         thinking_config: NotGivenOr[types.ThinkingConfigOrDict] = NOT_GIVEN,
+        gemini_tools: NotGivenOr[list[LLMTool]] = NOT_GIVEN,
```

Comment:
I'm not sure about this pattern, feels like this should be inside `Agent.tools` instead

Reply:
how do we handle the different types by diff vendors?

I'll be glad to back this change out if we are thinking about an alternative path